src/models/movie.model.tsx

```diff
@@ -0,0 +1,9 @@
+export interface MovieProps {
```

Comment:
This is a `Movie` type, not a `MovieProps` so please rename it. Props imply component params definition. You can also add an additional MovieProps which would just have Movie, e.g.
```
export interface MovieProps { 
  movie: Movie;
}
```

Reply:
Did you think I would do it like this?
export interface MovieProps { 
  movie: Movie;
}

export interface Movie {
id...
}

Reply:
Yes, but move MovieProps to the component itself.